portal/src/main/webapp/js/src/cbio-util.js

```diff
@@ -468,13 +468,12 @@ cbio.util = (function() {
     function baseMutationMapperOpts() {
         return {
             proxy: {
-                // default pdb proxy are now configured for a separate pdb data source
-                // this is for backward compatibility
+                // configure for the separate pdb-annotation API
```

Comment:
All data retrieving classes named as Proxy in the MutationMapper, not a good naming convention I guess. Those classes are actually API clients. And this line is required to configure the client to get the data from the new (separate) PDB data service.